zeth-contracts/contracts/BaseMixer.sol

```diff
@@ -244,6 +241,25 @@ contract BaseMixer is MerkleTreeMiMC7, ERC223ReceivingContract {
         hsig = bytes32(high_bits + uint(hsig_bytes));
     }
 
+    // This function is used to reassemble the commitment given the commitment
+    // index [0, jsOut[ and the primary_inputs. To do so, we extract the
+    // remaining bits of the commitment from the residual field element(S) and
+    // combine them with the commitment field element.
+    function assemble_commitment(
```

Comment:
I would rename this function to something like "retrieve_commitment" because it doesn't "assemble" it anymore.

Is this function actually still needed?

Reply:
Function removed.
Renaming `assemble_commitments_and_append_to_state` to `append_commitments_to_state`